v2/manager.go

```diff
@@ -581,13 +605,12 @@ func (c *Manager) EventChan() (<-chan Event, <-chan error) {
 	errCh := make(chan error)
```

Comment:
IMO, one buffer size should be good here. If no one reads the channel, the waitForEvents will be blocked in some condition.